adapters/resetdigital/resetdigital.go

```diff
@@ -0,0 +1,274 @@
+	Site resetDigitalRequestSite  `json:"site"`
+	Imps []resetDigitalRequesImps `json:"imps"`
+}
+
+type resetDigitalRequestSite struct {
+	Domain   string `json:"domain"`
+	Referrer string `json:"referrer"`
+}
+
+type resetDigitalRequesImps struct {
+	ForceBid bool `json:"force_bid"`
+	ZoneID   struct {
+		PlacementID string `json:"placementId"`
+	} `json:"zone_id"`
+	BidID string `json:"bid_id"`
+	ImpID string `json:"imp_id"`
+	Ext   struct {
+		Gpid string `json:"gpid"`
+	} `json:"ext"`
+	Sizes      [][]int64 `json:"sizes"`
+	MediaTypes struct {
+		Banner struct {
+			Sizes [][]int64 `json:"sizes"`
+		} `json:"banner"`
+		Video struct {
+			Sizes [][]int64 `json:"sizes"`
+		} `json:"video"`
+	} `json:"media_types"`
+}
+
+func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server config.Server) (adapters.Bidder, error) {
+	template, err := template.New("endpointTemplate").Parse(config.Endpoint)
+	if err != nil {
+		return nil, fmt.Errorf("unable to parse endpoint url template: %v", err)
+	}
+
+	bidder := &adapter{
+		endpoint: template,
+	}
+
+	return bidder, nil
+}
+func getHeaders(request *openrtb2.BidRequest) http.Header {
+	headers := http.Header{}
+
+	if request.Device != nil && request.Site != nil {
+		addNonEmptyHeaders(&headers, map[string]string{
+			"Referer":         request.Site.Page,
+			"Accept-Language": request.Device.Language,
+			"User-Agent":      request.Device.UA,
+			"X-Forwarded-For": request.Device.IP,
+			"X-Real-Ip":       request.Device.IP,
+			"Content-Type":    "application/json;charset=utf-8",
+			"Accept":          "application/json",
+		})
+	}
+
+	return headers
+}
+func addNonEmptyHeaders(headers *http.Header, headerValues map[string]string) {
+	for key, value := range headerValues {
+		if len(value) > 0 {
+			headers.Add(key, value)
+		}
+	}
+}
+
+func getReferer(request *openrtb2.BidRequest) string {
+	if request.Site == nil {
+		return ""
+	}
+
+	return request.Site.Domain
+}
+
+func getCurrency(request *openrtb2.BidRequest) string {
+	if len(request.Cur) == 0 {
+		return "USD"
+	}
+
+	return request.Cur[0]
+}
+
+func (a *adapter) MakeRequests(requestData *openrtb2.BidRequest, requestInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	var (
+		requests []*adapters.RequestData
+		errors   []error
+	)
+
+	referer := getReferer(requestData)
+	currency := getCurrency(requestData)
+
+	if referer == currency {
+		return nil, nil
+	}
+
+	for i := range requestData.Imp {
+		imp := requestData.Imp[i]
+		bidType, err := getBidType(imp)
+
+		if err != nil {
+			errors = append(errors, err)
+			continue
+		}
+
+		splittedRequestData := processDataFromRequest(requestData, imp, bidType)
+
+		requestBody, err := json.Marshal(splittedRequestData)
+		if err != nil {
+			errors = append(errors, err)
+			continue
+		}
+
+		requests = append(requests, &adapters.RequestData{
+			Method:  "POST",
+			Uri:     a.endpointUri,
+			Body:    requestBody,
+			Headers: getHeaders(requestData),
+		})
```

Comment:
requesting to merge `Prebid:master` changes into `bruno-siira:master` and  update `MakeRequests` to return ImpIDs as  mentioned below

https://github.com/prebid/prebid-server/blob/763551963dd3208dde6f9bf35b31ef76cc83814f/adapters/pubmatic/pubmatic.go#L175-L182

Reply:
Yes, please.

Reply:
These headers are added inside the getHeaders method before.